doc/user_guide/getting_started.rst

```diff
@@ -221,8 +221,61 @@ in the Signal2D subclass.
     >>> im
     <Signal2D, title: , dimensions: (30|20, 10)>
 
-Note the HyperSpy rearranges the axes position to match the following pattern:
-(navigatons axis 0,..., navigation axis n|signal axis 0,..., signal axis n).
+Note the HyperSpy rearranges the axes when compared to the array order. It does
+so due to a couple of things that all contribute:
+
+
+- Depending how the array is arranged, some axes are faster to iterate than
+  others. Consider an example of a book as the dataset in question. It is
+  trivially simple to look at letters in a line, and then lines down the page,
+  and finally pages in the whole book.  However if your words are written
+  vertically, it's slightly inconvenient to read top-down (the lines are still
+  horizontal, it's just the meaning that's vertical!). It's just awful if
+  every letter is on a different page, and for every word you have to turn 5-6
+  pages. Exactly the same idea applies here - in order to iterate through the
+  data (most often for plotting, but applies for any other operation too), you
+  want to keep it ordered for "fast access".
+
+- In python (more explicitly `numpy`) the "fast axes order" is the C-order.
+  This means that the **last** axis of a numpy array is fastest to iterate over
+  (i.e. the lines in the book). There are alternatives in other languages,
+  namely the F (for Fortran)-order, where it's the reverse - the first axis of
+  an array is the fastest to iterate over. Usually in all implementations
+  iterating over some axis in the middle is slow, as you have to "pick the same
+  line from all pages", or something similar.
+
+- Most datasets are recorded in the C-order. Taking EDS / EELS map as an
+  example, the dataset has three dimensions/axes - the fast, and slow
+  real-space, and the energy channels. Thinking how the microscope works,
+  it's obvious that the energy axis is the fastest, then the x, and finally
+  the y is the slow axis. To record it in C-order, it is stored in an array
+  (y, x, E). If this was a SPED dataset, it would be recorded in (y_real,
+  x_real, y_recip, x_recip) with real and reciprocal spaces respectively.
+
+- In HyperSpy we want to order these things for humans. **Which means x axis
```

Comment:
In HyperSpy we want to order these things for humans, namely **the x axis goes before y**.